### PR TITLE
fix path to globals

### PR DESCRIPTION
**Pull request by stephenwaite**

### CODE REVIEW

**Comment by stephenwaite** [comment]
didn't seem to work, is there a $GLOBALS path should use instead or maybe 
    include_once("{$GLOBALS['srcdir']}/../interface/globals.php");

**Comment by bradymiller** [comment]
For a library, is best to hard code(and using relative directory is not consistent since depends on the script that calls it; so should use dirname(__FILE__), which is path to the library):
``````
require_once(dirname(__FILE__) . "/../interface/globals.php");
``````
Good examples here:
https://github.com/openemr/openemr/blob/master/library/sql.inc#L23-L26

**Comment by bradymiller** [comment]
Note that can only use $GLOBALS['srcdir'] if you are sure interface/globals has been called since that script sets that up. So ok to use it in the require_once of billing.inc in this library.

**Comment by stephenwaite** [comment]
thanks Brady, submitted another try

File: library/api.inc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 //our api for 3rd party developers
-include_once("../../globals.php");
+include_once(dirname(__FILE__) . "../interface/globals.php");
```
Comment (bradymiller) [inline library/api.inc]: I think you'll need a slash at the beginning:
``````
include_once(dirname(__FILE__) . "/../interface/globals.php");

Comment (bradymiller) [inline library/api.inc]: also, make this the the one below a require_once

Comment (stephenwaite) [inline library/api.inc]: good pts

**Comment by bradymiller** [comment]
looks good. thanks for the fix! -brady

**Comment by stephenwaite** [comment]
you're welcome, trying to track down all of the pesky PHP Warnings, thanks for all of the help with the code

**Pull request merged**